packages/react-variant-view/src/component/frequencyTable/FrequencyCell.tsx

```diff
@@ -0,0 +1,58 @@
+// this component comes from signal
```

Comment:
We should either move this to `cbioportal-frontend-commons` or the depending implementation into signal.

Reply:
this is moved to `cbioportal-frontend-commons`